docs/guides/result_storage.mdx

```diff
@@ -114,3 +104,18 @@ await dataset.pushData([{ foo: 'bar2', col2: 'val2' }, { col3: 123 }]);
 ```
 
 To see how to use the dataset to store crawler results, see the [Cheerio Crawler](../examples/cheerio-crawler) example.
+
+## Cleaning up the storages
+
+Default storages are not purged before the crawler starts if not specified otherwise. If you want to start the run with the empty default storages, you should first <ApiLink to="core/class/Configuration#getStorageClient">get</ApiLink> the default <ApiLink to="types/interface/StorageClient">`StorageClient`</ApiLink> instance and then use the <ApiLink to="types/interface/StorageClient#purge">`purge()`</ApiLink> function before running the crawler as follows:
+
+```javascript
+import { Configuration } from '@crawlee/core';
+//                   ... from '@crawlee/cheerio';
+//                   ... from '@crawlee/playwright';
+//                   ... from '@crawlee/puppeteer';
+
+await Configuration.getStorageClient().purge();
```

Comment:
we should probably add a `purgeStorage` function that will do this so users dont have to write such a weird line :]